server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementScanner.java

```diff
@@ -52,9 +59,16 @@ public class TabletManagementScanner implements ClosableIterator<TabletManagemen
     // scan over metadata table, looking for tablets in the wrong state based on the live servers
     // and online tables
     try {
-      mdScanner = context.createBatchScanner(tableName, Authorizations.EMPTY, 8);
+      Locations locs = context.tableOperations().locate(tableName, ALL_TABLETS_RANGE);
```

Comment:
This call will clear the metadata location cache, which could impact other operations in the manager.  Could scan instead for this information, or maybe the impl of locat should scan instead of scanning via clearing the cache. Alternatively could use Suppliers.memoizeWithExpiration to not compute the numThreads too often.

Reply:
Another alternative is to go to the tablet location cache directly w/o clearing it.  Could just map tablets w/o a location to the same thing for counting purposes and not wait for tablets to have locations to compute the num threads.

Reply:
Thanks, I forgot that calling tops.locate clears the cache.

Reply:
Updated to use tablet location cache directly in 2b83d93